docs/source/usage/new_user_guide.rst

```diff
@@ -0,0 +1,90 @@
+New User Guide
+==============
+
+Prerequisites
+-------------
+
+The first step is to gain access to an ESI installation. After you do, the ESI operator will send you credentials and instructions regarding how to access your account.
+
+Afterwards you'll need to install the ESI clients. It is recommended that you do so within a python virtual environment:
+
+  .. prompt:: bash $
+
+    python3 -mvenv .venv
+    source .venv/bin/activate
+
+To install the ESI clients, run the following:
+
+  .. prompt:: bash (.venv)$
+
+    pip install python-esiclient
+    pip install python-esileapclient
+
+Viewing Your Nodes
+------------------
+
+The ESI operator will have given you access to nodes. In order to view them, you can run one of the following:
+
+  .. prompt:: bash (.venv)$
+
+    openstack baremetal node list          # Ironic's node list command
+    openstack esi node list                # ESI's node list command with leasing information
+    openstack esi node network list        # ESI's node list command with node network information
+
+To view additional details about a node, run:
+
+  .. prompt:: bash (.venv)$
+
+    openstack baremetal node show <node ID>   # The node ID can be its name or UUID
+
+Node Provisioning States (and what they mean)
+---------------------------------------------
+
+``openstack baremetal node list`` and ``openstack esi node list`` both display a node's provisioning state. These states determine what actions can be performed upon a node.
+
+* **manage**: node is configured and ready to be cleaned
+* **available**: node has been cleaned and is ready for deployment 
+* **active**: node has been deployed
+* **cleaning**: node is in the process of being cleaned before being returned to the **available** state
+
+There are also transitionary states that indicate a node is in the process of moving from one state to another.
+
+Configuring Your Networks
+-------------------------
+
+Your ESI installation may have networks available for use. However you may also want to configure private networks, accessible only by your project. For details on the options available, read the `documentation regarding private networks`_ and talk to your ESI operator.
+
+Configuring and Deploying Your Nodes
```

Comment:
Is there any place in the doc that mentions how to log in the node? Like what is the ip address of the node and which ssh key you can use

Reply:
I've just added a small explanatory blurb - let me know if that works!